tests/qchem/of_tests/test_convert.py

```diff
@@ -987,15 +999,134 @@ def test_excited_configurations_error(excitation):
         _ = qchem.convert._excited_configurations(2, 4, excitation)
 
 
-def test_fail_import_pyscf(monkeypatch):
-    """Test if an ImportError is raised when pyscf is requested but not installed."""
```

Comment:
Why was this test removed?

Reply:
We just check the string representation of the object to decide what type they are. Otherwise, we have to import several kind of libraries inside the function.